tests/Doctrine/Tests/ORM/Mapping/AbstractMappingDriverTest.php

```diff
@@ -278,13 +278,6 @@ public function testFieldIsNullableByType(): void
 
         // Explicit Not Nullable
         $this->assertFalse($class->isNullable('username'));
-
-        // Join table Nullable
-        $this->assertFalse($class->getAssociationMapping('email')['joinColumns'][0]['nullable']);
-        $this->assertEquals(CmsEmail::class, $class->getAssociationMapping('email')['targetEntity']);
-
-        $this->assertTrue($class->getAssociationMapping('mainEmail')['joinColumns'][0]['nullable']);
-        $this->assertEquals(CmsEmail::class, $class->getAssociationMapping('mainEmail')['targetEntity']);
```

Comment:
I believe `targetEntity` tests should stay. My method naming is lacking here as it suggests only checks for `nullable`, while in fact there are 2 checks for `targetEntity` also. Probably method name change would be good?